qiskit/optimization/algorithms/admm_optimizer.py

```diff
@@ -68,25 +79,31 @@ def __init__(self, rho_initial=10000, factor_c=100000, beta=1000, max_iter=10, t
         self.factor_c = factor_c
         self.beta = beta
         self.rho_initial = rho_initial
-        self.qubo_solver = qubo_solver if qubo_solver is not None else CplexOptimizer()
-        self.continuous_solver = continuous_solver if continuous_solver is not None else CplexOptimizer()
+        self.qubo_optimizer = qubo_optimizer if qubo_optimizer is not None else CplexOptimizer()
+        self.continuous_optimizer = continuous_optimizer if continuous_optimizer is not None \
+            else CplexOptimizer()
```

Comment:
There is a shorthand notation for these `if .. is not None else` statements:
```suggestion
        self.qubo_optimizer = qubo_optimizer or CplexOptimizer()
        self.continuous_optimizer = continuous_optimizer or CplexOptimizer()
```